FieldOpt/Runner/runners/oneoff_runner.cpp

```diff
@@ -22,6 +22,7 @@ void OneOffRunner::Execute()
     applyWellPositionFromArguments();
     EvaluateBaseModel();
     // Write objective function value to file
+    model_->wellCost(settings_->optimizer()->objective());
     Utilities::FileHandling::WriteLineToFile(QString("%1").arg(objective_function_->value()),
```

Comment:
I know this isn't your code Brage, but when on earth did this slip in? What's the point? It's already written to the case log...

Reply:
Line 25 or 26? I have no idea, I no touchy line 26